src/XAMLMarkupExtensions.csproj

```diff
@@ -43,18 +44,8 @@
 	</Choose>
 
 	<ItemGroup>
-		<Reference Include="PresentationCore" />
```

Comment:
The removed references are added by changing the SDK to `Microsoft.NET.Sdk.WindowsDesktop`